analyticsclient/constants/education_level.py

```diff
@@ -7,3 +7,4 @@
 BACHELORS = 'bachelors'
 MASTERS = 'masters'
 DOCTORATE = 'doctorate'
+UNKNOWN = 'unknown'
```

Comment:
The other education level constants are tested in https://github.com/open-craft/edx-analytics-data-api-client/blob/user-profile-api/analyticsclient/tests/test_helpers.py#L21-L30, so it would make sense to add a test for `UNKNOWN` (although I don't really see much value in these sort of tests).